src/Pages/LandingPage/LandingPage.css

```diff
@@ -0,0 +1,186 @@
+.landing__page__container{
+    height:100vh;
+    width:100vw;
+    background-color: #ffffff;
+    display: flex;
+    justify-content:start;
+    align-items: center;
+    overflow: hidden;
+}
+.landing__page--text{
+    height:100%;
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    margin-left:10rem;
+}
+.landing__page__logo--text{
+    font-size: 12rem;
+    color:#fca311;
+}
+.landing__page__heading{
+    margin:1rem 0rem;
+}
+.landing__page__heading--text{
+    font-size: 6rem;
+    color:#000000;
+}
+.landing__page__description--text{
+    font-size: 3rem;
+    margin: 0rem 0rem 0rem 0rem;
```

Comment:
Why are you not giving just ` margin: 0; ` here? It would also work

Reply:
Forgot to remove thanks!